docker/mongodb/source/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+### This script relies on provided keytab file which will be
+### be mounted to /etc/secrets area
+if [ -f /tmp/krb ]; then
+  export keytab=/tmp/krb/cmsweb.keytab
```

Comment:
I'm not sure I understand logic here. The comment says it relies on /etc/secrets area while you use /tmp/rkb, and I checked k8s manifest which also provides keytab file in /tmp/krb. Is there are any reason why it should be in /tmp/krb and not in /etc/krb for instance?

Reply:
@vkuznet, sorry that is a wrong path in the comment. I will modify it.